examples/python-interop/README.md

```diff
@@ -0,0 +1,62 @@
+# Python Interop
+
+This is an example of calling Roc code from [Python](https://www.python.org/).
+
+## Installation
+
+The following was tested on NixOS, with Python 3.10, clang 13.0.1, gcc 11.3.0 but this should work on with most recent python3 and clang versions on most modern Linux and MacOS.\
+Of course you're welcome to test on your machine and tell me if you ran into any issues or limitations.
```

Comment:
Someone coming into the repo fresh won't know who "me" is - maybe include a link to DM you on Zulip? 😄

Reply:
Right, added a link, commited below